src/instaskip/route_with_path_process.clj

```diff
@@ -1,31 +1,43 @@
 (ns instaskip.route-with-path-process
-  [:require [instaskip.innkeeper-client :as ik]])
+  [:require [instaskip.innkeeper-client :as ik]
+            [cats.core :as c]
+            [cats.builtin]
+            [cats.monad.exception :as exc]
+            [cats.context :as ctx]])
 
 (defn- host->host-id [hosts-to-ids host]
 
-  (let [id (hosts-to-ids host)]
-    (if (nil? id)
-      (throw (Exception. (str "Host " host " not defined in innkeeper!"))))
-    id))
+  (exc/try-on
+    (let [id (hosts-to-ids host)]
```

Comment:
made it better... good suggestion! the change will come with the next pull request